### ISSUE TITLE
Add DeepAlpha - AI blockchain trading bot

### ISSUE THREAD

**Issue by stefanoviana** [yjjnls/awesome-blockchain#89]
Suggesting [DeepAlpha](https://github.com/stefanoviana/deepalpha) for **Applications > Trading**.

AI crypto trading bot for Bybit & Binance with 70.9% accuracy, LightGBM + XGBoost ensemble, 72 ML features. MIT licensed.
